internal/http/services/owncloud/ocs/response/response.go

```diff
@@ -45,6 +45,17 @@ type Response struct {
 	OCS *Payload `json:"ocs"`
 }
 
+// NewResponse returns an empty response
+func NewResponse() Response {
```

Comment:
this is only used in one place. Do we need such abstraction instead of having a struct literal? Its usage is in `sharing/shares/pending.go:L271`

Reply:
Removed in https://github.com/cs3org/reva/pull/2383